### ISSUE TITLE
[1a7575d][f08][171202-2234] training is too time-consuming

### ISSUE THREAD

**Issue by davisjazz** [davisjazz/SDC_PRJ03_Cloning#4]
**# REF :** 1a7575d

**# SCENARIO :**
1.  [win10][cmd] activate keras 
2.  [win10][cmd] python f08_model_171202-2234.py

**# EXPECTED RESULT:**   
- successful training and in acceptabe timeframe
- generation of the model.h5 file, ready to be tested in the simulator

**# CURRENT RESULT:**   
`12265/12266 [============================>.] - ETA: 3s - loss: 5.4239Epoch 00001: val_loss improved from inf to 0.05352, saving model to C:/log/model/ckpt_W_171202_2350_01_0.05.hdf5   
12266/12266 [==============================] - 38345s 3s/step - loss: 5.4234 - val_loss: 0.0535
Epoch 2/30   
12265/12266 [============================>.] - ETA: 0s - loss: 0.0528Epoch 00002: val_loss improved from 0.05352 to 0.05348, saving model to C:/log/model/ckpt_W_171202_2350_02_0.05.hdf5   
12266/12266 [==============================] - 10812s 881ms/step - loss: 0.0528 - val_loss: 0.0535
Epoch 3/30   
12265/12266 [============================>.] - ETA: 0s - loss: 0.0529Epoch 00003: val_loss did not improve   
12266/12266 [==============================] - 8135s 663ms/step - loss: 0.0529 - val_loss: 0.0535
Epoch 4/30   
 2236/12266 [====>.........................] - ETA: 1:36:13 - loss: 0.05302017-12-03 16:06:53.317091: E c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\cuda\cuda_driver.cc:1067] could not synchronize on CUDA context: CUDA_ERROR_LAUNCH_FAILED :: No stack trace available   
2017-12-03 16:06:53.317464: F c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\core\common_runtime\gpu\gpu_util.cc:370] GPU sync failed`   

It failed because I ran another 'python model.py' in another cmd_prompt.
There was a risk but it is ok because **the result would be wrong due to ',' vs '.' issue** and **the data need to be resized before running again the training**

**# ENVIRONMENT :**
- platform : win10-64
- GPU: Nvidia Geforce 840M
- tensorflow-gpu 1.1.0 <pip>
- Keras 2.1.1 <pip>
- python 3.5.2 0
- numpy 1.12.1 <pip>
- opencv3 3.1.0 py35_0 menpo
- scikit-learn 0.19.0 np112py35_0
- scipy 1.0.0 <pip>
 ---
- h5py 2.7.1 <pip>
- hdf5 1.10.1 h98b8871_1 anaconda
- eventlet 0.21.0 <pip>
- Flask 0.12.2 <pip>
- greenlet 0.4.12 <pip>
- h5py 2.7.1 <pip>
- hdf5 1.10.1 h98b8871_1 anaconda
- imageio 2.1.2 <pip>
